hack/utils.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -x
+
+function running_from_skipper() {
+   # The SKIPPER_UID environment variable is an indication that we are running on a skipper container.
+   [ -n "${SKIPPER_UID+x}" ]
```

Comment:
We check `SKIPPER_USERNAME` on the Makefile. Could you align them? I don't care which one :) 
https://github.com/openshift/assisted-service/blob/da115d03cbd81b752a3777f4bd5d34459beaf089/Makefile#L127-L130